storage/redis/redis_scheduler_event_storage.go

```diff
@@ -0,0 +1,79 @@
+// maestro
+// https://github.com/topfreegames/maestro
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2017 Top Free Games <backend@tfgco.com>
+
+package redis
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/go-redis/redis"
+	"github.com/topfreegames/extensions/redis/interfaces"
+	"github.com/topfreegames/maestro/models"
+)
+
+type RedisSchedulerEventStorage struct {
```

Comment:
The interface is a little different from the implementation. We check if the struct implements it at compile-time by adding lines like `var _ SchedulerEventStorage = (*RedisSchedulerEventStorage)(nil)`. [Like we do on next](https://github.com/topfreegames/maestro/blob/next/internal/adapters/runtime/kubernetes/kubernetes.go#L30).